test/e2e/common/host_path.go

```diff
@@ -276,3 +189,67 @@ func testPodWithHostVol(path string, source *v1.HostPathVolumeSource) *v1.Pod {
 		},
 	}
 }
+
+func testExistingSubPath(f *framework.Framework, fileTest bool) {
+	framework.SkipUnlessSSHKeyPresent()
+
+	var subPath string
+	var command string
+	var filePathInReader string
+
+	retryDuration := 180
```

Comment:
please add a consts section at the top of the file.

Reply:
Thanks for yor review, @timothysc 
That is a nice point, retryDuration seems to be defined at several places in this file.
It is nice to merge them into a single const value.
OK, I will do that later with another thing.